Src/Installer/Dasher.wxs

```diff
@@ -269,13 +267,13 @@
       <ComponentRef Id="Colours"/>
       <ComponentRef Id="Control"/>
       <ComponentRef Id="Settings"/>				
-				<ComponentRef Id="Help"/>
```

Comment:
Same here (comment it out instead of deleting and reference the issue)

```
<!-- Commenting this out for Appveyor CI to work: see /dasher/issues/148
  <ComponentRef Id="Help"/>
-->